client/layout/masterbar/masterbar-cart-button-style.scss

```diff
@@ -0,0 +1,28 @@
+.masterbar-cart-button {
+	display: flex;
+}
+
+.masterbar-cart-button button {
+	border-radius: 2px;
+	border: 1px solid var( --studio-gray-50 );
+}
+
+.masterbar-cart-button__count-wrapper {
+	position: relative;
+}
+
+.masterbar-cart-button__count-container {
+	display: inline-block;
+	padding: 1px 6px;
```

Comment:
I'm using the same padding that we currently use for the popover cart, and I'm hesitant to change it. 

<img width="224" alt="Screen Shot 2021-11-30 at 6 29 21 PM" src="https://user-images.githubusercontent.com/2036909/144144635-70f53925-88ff-48d2-aaf4-d2fc81178452.png">


Also I think it's very unlikely (at the moment) for anyone to have double digit cart items, so IMO this might be mostly academic.

Reply:
Your ratioinale makes a lot of sense - let's keep it as-is. I must have been in an overly "try to break something" mode here 😅